app/controllers/reservations_controller.rb

```diff
@@ -4,14 +4,28 @@ class ReservationsController < ApplicationController
   before_action :set_reservation, only: %i[show edit update destroy]
   before_action :authenticate_user!
 
+  def find_reservations_by_user
```

Comment:
It looks like you use [rubocop](https://github.com/MStodolkiewicz/Monte_Cinema/blob/main/.rubocop.yml), but the syntax violates the defined rules by you.

Do you need help with configuring rubocop plugin for your IDE?

```
Offenses:

app/controllers/reservations_controller.rb:8:47: C: [Correctable] Layout/TrailingWhitespace: Trailing whitespace detected.
    @reservations = Reservation.where(user_id: 
                                              ^
app/controllers/reservations_controller.rb:10:5: C: [Correctable] Layout/MultilineMethodCallIndentation: Align .joins with .where on line 8.
    .joins(:seance).includes([:user, seance:[:movie]])
    ^^^^^^
app/controllers/reservations_controller.rb:10:38: C: [Correctable] Style/HashAsLastArrayItem: Wrap hash in { and }.
    .joins(:seance).includes([:user, seance:[:movie]])
                                     ^^^^^^^^^^^^^^^
app/controllers/reservations_controller.rb:10:44: C: [Correctable] Layout/SpaceAfterColon: Space missing after colon.
    .joins(:seance).includes([:user, seance:[:movie]])
```

Reply:
No, sorry, I had to stash this for a second and I probably run rubocop before popping it.
Rubocop returned offenses now so it's fixed :).